editor/scene_tree_dock.cpp

```diff
@@ -3412,11 +3431,13 @@ SceneTreeDock::SceneTreeDock(Node *p_scene_root, EditorSelection *p_editor_selec
 
 	button_tree_menu = memnew(MenuButton);
 	button_tree_menu->set_flat(true);
-	button_tree_menu->connect("about_to_popup", callable_mp(this, &SceneTreeDock::_update_tree_menu));
 	filter_hbc->add_child(button_tree_menu);
 
 	PopupMenu *tree_menu = button_tree_menu->get_popup();
-	tree_menu->add_check_item(TTR("Auto Expand to Selected"), TOOL_AUTO_EXPAND);
+	tree_menu->set_hide_on_checkable_item_selection(false);
```

Comment:
If we add actions to this menu (e.g. #65932), this will make it not disappear on button click, which is not really desired. Unless it only affects checkboxes, then ok.